### DISCUSSION TITLE
Catalog Resource is denied

### DISCUSSION THREAD

**Discussion by aisom02** [saleweaver/python-amazon-sp-api#569] [General]
Hello, I have been using feeds and reports functions in the SP-API for quite some time. When I try and create a request to list items using catalogs I get sp_api.base.exceptions.SellingApiForbiddenException: [{'message': 'Access to requested resource is denied.', 'code': 'Unauthorized', 'details': ''}]. Are there specific permissions I need to set within my iam user/role and or app profile on the seller portal that would be different to using the reports or inventory api? I have gone through and checked permissions and don't see anything that would conflict with calling with the catalog function.

**Comment by chacin04** [comment]
resolved ?
